src/tests/dictionaryConcepts/container/DictionaryConcepts.test.jsx

```diff
@@ -58,12 +65,13 @@ describe('Test suite for dictionary concepts components', () => {
       filteredSources: ['CIEL'],
       loading: false,
     };
-    const wrapper = mount(<Router>
+    const wrapper = mount(<Provider store={store}><Router>
       <DictionaryConcepts {...props} />
-    </Router>);
+    </Router></Provider>);
     const event = { target: { name: 'searchInput', value: 'ciel' } };
     wrapper.find('#search-concept').simulate('change', event);
-    wrapper.find('form').simulate('submit', {
+    expect(wrapper.find('form.searchConcepts')).toHaveLength(1);
+    wrapper.find('.searchConcepts').simulate('submit', {
```

Comment:
I did not change anything here. This was not within my scope of work

Reply:
I see you changed something there. If it's not in your scope. Don't edit anything. Otherwise, this test is incomplete.

Reply:
@dondrzzy  The reason I decided to change from 'form' to '.searchConcepts' is to fix this test since it was failing. But I could not introduce more fixes, since that is beyond this scope.